elide-graphql/src/main/java/com/yahoo/elide/graphql/ModelBuilder.java

```diff
@@ -65,6 +65,8 @@
     private Map<Class<?>, GraphQLObjectType> connectionObjectRegistry;
     private Set<Class<?>> excludedEntities;
 
+    private HashMap<String, GraphQLInputType> convertedInputs = new HashMap<>();
```

Comment:
Why do we need this?

Reply:
We want to reuse the same types. If we blindly call `newObject()` the updated GraphQL fails; it doesn't allow us to have multiple types with the same name (even if those types are actually identical).

Overall, I think this is beneficial. In fixing this I have uncovered several additional bugs with how we were re-writing before.